jormungandr-watchdog/src/service/mod.rs

```diff
@@ -145,7 +145,7 @@ impl<T: Service> ServiceState<T> {
     /// this from there one can "borrow" the settings or clone the reader
     ///
     /// [`SettingsReader`]: ./struct.SettingsReader.html
-    pub fn settings(&self) -> &SettingsReader<T::Settings> {
+    pub fn settings(&mut self) -> &SettingsReader<T::Settings> {
```

Comment:
do we still need the mut here?

Reply:
Indeed we do not. Missing change from bringing it back to original form.